portfolio/src/main/java/com/google/sps/servlets/DeleteResponseServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting tasks. */
+@WebServlet("/delete-response")
+public class DeleteResponseServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
You've met the Personal Portfolio requirements so this isn't mandatory. However, I think you can make these changes fairly easily and it would make your portfolio cleaner and in-line with best practices.